fontforgeexe/cvimportdlg.c

```diff
@@ -362,14 +470,38 @@ return( oldflags );
     while ( !done )
 	GDrawProcessOneEvent(NULL);
 
-    /* This dlg can't be cancelled */
-    oldflags = 0;
-    if ( GGadgetIsChecked(gcd[cd_k].ret) )
-	oldflags |= sf_correctdir;
-    if ( GGadgetIsChecked(gcd[he_k].ret) )
-	oldflags |= sf_handle_eraser;
+    ip->correct_direction = GGadgetIsChecked(gcd[cd_k].ret);
+    ip->erasers = GGadgetIsChecked(gcd[he_k].ret);
+    ip->simplify = GGadgetIsChecked(gcd[si_k].ret);
+    ip->clip = GGadgetIsChecked(gcd[cl_k].ret);
+    ip->scale = GGadgetIsChecked(gcd[sc_k].ret);
+    ip->default_joinlimit = GetReal8(gw,CID_JoinLimitVal,_("Default Join Limit (PS/EPS/SVG):"),&err);
+    if ( err ) {
+	ip->default_joinlimit = JLIMIT_INHERITED;
+	err = false;
+    }
+    ip->accuracy_target = GetReal8(gw,CID_AccTar,_("Accuracy Target:"),&err);
+    if ( err )
+	ip->accuracy_target = 0.25;
+    ip->show_always = GGadgetIsChecked(gcd[al_k].ret);
+
     GDrawDestroyWindow(gw);
-return( oldflags );
+}
+
+static void ShowImportOptions(ImportParams *ip, int shown, int type) {
+    if ( type==0 ) {
```

Comment:
Not a fan of these magic numbers

Reply:
I've since thought that a unified flag interface for both the state and the argument would be cleaner, but that can wait for a later straightening-up.

Reply:
I did this anyway